popSimManu.tex

```diff
@@ -277,16 +278,16 @@ \section*{Results}
 extremely compact, and can be processed efficiently using the \tskit library
 \citep{kelleher2016efficient,kelleher2018efficient}. Currently,
 \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
-as the default simulation engine. We plan to include other simulation
-engines to support scenarios that cannot be modeled under the coalescent framework,
-and have already implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} as
-an additional backend.
+as the default simulation engine.
+We have implemented \texttt{SLiM} \citep{haller2019tree,haller2019slim} 
+as an alternative backend,
+to allow simulation of processes that cannot be modeled under the coalescent.
```

Comment:
what if we make this a positive thing and say "to allow simulation of processes that can only be modeled forward in time"?

Reply:
well, that's not quite the same thing? I vote to keep this.